utils/cmdline.py

```diff
@@ -27,7 +27,7 @@ def get_cache_fn(fn):
     an impact on the operation of mkt - at worst it will run slower."""
     global cache_dir
     if cache_dir is None:
-        cache_dir = os.path.expanduser("~/.cache/mellanox/mkt/")
+        cache_dir = '/images/ztiffany/.cache/mellanox/mkt/'
```

Comment:
the home dir is insufficient to hold these caches,
Is there a way to point it somewhere else?

Reply:
.ccache is a symlink

Reply:
Makes sense